types/intl-locales-supported/intl-locales-supported-tests.ts

```diff
@@ -0,0 +1,4 @@
+import areIntlLocalesSupported from "intl-locales-supported";
```

Comment:
This should be `import areIntLocalesSupported = require("intl-locales-supported");`

Reply:
done